matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/login/DefaultLoginWizard.kt

```diff
@@ -62,14 +64,16 @@ internal class DefaultLoginWizard(
                     address = login,
                     password = password,
                     deviceDisplayName = initialDeviceName,
-                    deviceId = deviceId
+                    deviceId = deviceId,
+                    enableRefreshTokenAuth = matrixConfiguration.enableRefreshTokenAuth
```

Comment:
I would not introduce `MatrixConfiguration.enableRefreshTokenAuth`, which is used as a global configuration of the SDK, but rather add a boolean to the API `login` (and probably some other API) so that the caller can choose to enable refresh token facility or not per session.

Reply:
This also reminds me I never added `refresh_token` param to the registration request.